### ISSUE TITLE
Using customizer to render panels in a specific div is broken

### ISSUE THREAD

**Issue by ahmedkotb** [OfficeDev/office-ui-fabric-react#5696]
### Bug Report 
 
- __Package version(s)__:  starting from 6.37.0
- __Browser and OS versions__: Chrome/windows10
 
#### Priorities and help requested:

Are you willing to submit a PR to fix? Yes
Requested priority: High

#### Describe the issue:
 
the example in the Layer page to trap a panel into a certain div using the customizer component is not working any more (starting from version 6.37.0, was working fine on version 6.36.0)

 https://developer.microsoft.com/en-us/fabric#/components/layer
#### Actual behavior:
 
Panel is rendered outside of the specified layer
 
#### Expected behavior:
 
Panel should be rendered inside the specified div
 
### If applicable, please provide a codepen repro:

https://codepen.io/akotb/pen/NBbwzv
change the version to 6.37.0 and panel will be rendered outside the layer

**Comment by cliffkoh** [comment]
Reopening because I think this is a different issue with a very specific repro.

**Comment by papigers** [comment]
@ahmedkotb @cliffkoh I've also encountered this issue, I don't know exactly where is the problem in the `Customizer` (on which btw, I couldn't find any documentation other than in his source file), but I managed to trap the panel using the prop `layerProps`. I've edited your codepen to show an example:
https://codepen.io/papigers93/pen/qyVrRj
As you see, there's no need to use the `Customizer`, at all, and things are working as expected.

So this solves @ahmedkotb problem, but there's still might be a problem in `Customizer` you might want to fix.
